### ISSUE TITLE
Merging person order is the opposite of expected

### ISSUE THREAD

**Issue by mmomjian** [immich-app/immich#8335]
### The bug

When selecting to Merge a person, Immich defaults to use the currently selected person as the "master" (profile picture and name). I suggest that this is the opposite of expected behavior. When a user selects to "Merge" a person, I suggest they typically have a known (named) target for the Merge. It would make more sense that the default behavior is to Merge a person *into* an existing person, as opposed to the current behavior.

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

1.100.0

### Version of Immich Mobile App

1.100.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. View a person
2. Click Merge Person
3. the default behavior is to merge the pictures from someone else into this person
```

**Comment by almulder** [comment]
On mine I can select 1 person and then another and select what way to merge. The arrows between them. Pressing it swaps locations of who mergers into who. (I  just notice that option)

**Comment by pyorot** [comment]
generalises #6382 

a bit of a pain to fix tho because selecting people to merge can stack multiple people on the source side of the merge operation, which is more efficient but unintuitive enough to where nobody will think to do it

**Comment by C-Otto** [comment]
To work around this issue, you can edit the name of the "merge source" (victim) to the name of the "merge destination" (winner), prompting a dialog that you then need to confirm. This works like a charm (and I didn't even know about the "Merge people" options in the "..." menus).

**Comment by C-Otto** [comment]
I don't see how merging **multiple** already existing people (named or not) into one is a common use case. If we only allow merging two people (one into the other), we could easily switch the default order in the "merge people" dialog. This would also fix #6382.

**Comment by jrasm91** [comment]
There seem to be two common use cases:

1. Select a named person, go to the merge page, scroll down and select all duplicates
2. Scroll through the people page, find a duplicate, go to the merge page, selected a named person

#12601 solves this by:
- Keeping (1) as the default flow, which implies the original selection is the surviving person in the merge.
- Automatically swapping when (2) is detected (selected person has no name and first merge candidate has a name)

This still makes it possible to merge multiple people, even in case (2) as additional people can still be added later as additional merge sources.
